Fixed Quickstart (#1621)
near/docs: docs/2.develop/integrate/quickstart.md

  npx create-near-app@latest
```

Use the interactive menu to set up:
1. `A Near Gateway (Web App)`.
2. `NextJs + React`.

:::tip Using pnpm
While you can use our app with any package manager, we recommend you to skip the installation step and manually install the dependencies using `pnpm i`.
:::

Once the folder is ready - and all dependencies installed - you can start the development server using `pnpm`. 

```bash 
pnpm dev
```

Visit `http://localhost:3000` in your browser to view the dApp. Note that since the dApp uses NextJS the app might take longer to load the pages on first visit.

<details>
<summary> The app is not starting? </summary>

Make sure you are using **node >= v18**, you can easily switch versions using `nvm use 18`

</details>

---

## Landing Page

Once the app starts you will see the landing page, rendering a navigation bar that allows users to login using their NEAR wallet, and two pathways:

![img](/docs/assets/examples/hello-near-landing-page.png)
*Landing page of Hello NEAR Gateway*

> Go ahead and sign in with your NEAR account. If you don't have one, you will be able to create one in the moment.

Reviewer: ```suggestion
Go ahead and sign in with your NEAR account. If you don't have one, you can create one on the fly.
```

</details>

<hr class="subsection" />

### Navigation Bar & Login
The navigation bar implements buttons to `login` and `logout` users with their Near wallet.

The code for the navigation bar can be found at `./src/app/navigation.js`. The login and logout buttons are implemented by using the `logIn` and `logOut` methods from the wallet selector previously initialized:

<Github url="https://github.com/near/create-near-app/blob/master/templates/frontend/next/src/components/navigation.js" language="jsx" start="10" end="22" />

---

## Interacting with NEAR

Now that you understand how the landing page works, we can move to the `Near Integration` page, which retrieves a greeting from the [hello.near-examples.testnet](https://testnet.nearblocks.io/address/hello.near-examples.testnet) contract.

![img](/docs/assets/examples/hello-near-gateway.png)
*View of the `Near Integration` page*

Login if you haven't done it yet and you will see a simple form that allows you to store a greeting in the smart contract.

<hr class="subsection" />

### Under the Hood

> The interactions with NEAR are done by using the `useWallet` hook to retrieve the `signedAccountId` property, and `viewMethod` and `callMethod` methods from our `wallet selector`.

Reviewer: ```suggestion
Interactions with NEAR are done using the `useWallet` hook to retrieve both the `viewMethod` and `callMethod` methods and the `signedAccountId` property from the `wallet selector`.
```

<Github url="https://github.com/near/create-near-app/blob/master/templates/frontend/next/src/app/hello-near/page.js" language="jsx" start="12" end="32" />

> On load, the first `useEffect` hook will call the `get_greeting` method from the contract, and set the `greeting` state to the result.

Reviewer: ```suggestion
On load, the first `useEffect` hook will call the contract's `get_greeting` method and set the `greeting` state to the result.
```

> If the user is logged in, the `saveGreeting` method will call the `set_greeting` method from the contract, and set the `greeting` state to the result.

Reviewer: ```suggestion
If the user is logged in, the `saveGreeting` method will call the contract's `set_greeting` method and set the `greeting` state to the result.
```
